_includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/transient-reporting.adoc

```diff
@@ -0,0 +1,23 @@
+We need to create a query that extracts the fields we want from input sources.  Since we have defined the top element of the JSON as a String using the `VARCHAR` keyword,
+we can use the https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/scalar-functions/#extractjsonfield[ksqlDB EXTRACTJSONFIELD function] to extract the different values at a specified https://jsonpath.com/[JSONPath].
```

Comment:
JSONPath or JSON path? I don't have a strong preference, looks like this is intentional because it is linking out to webpage "JSONPath".